db/connection.js

```diff
@@ -1,12 +1,19 @@
-const { Pool } = require('pg');
-const ENV = process.env.NODE_ENV || 'development';
+const { Pool } = require("pg");
+const ENV = process.env.NODE_ENV || "development";
 
-require('dotenv').config({
+require("dotenv").config({
   path: `${__dirname}/../.env.${ENV}`,
 });
 
-if (!process.env.PGDATABASE) {
-  throw new Error('PGDATABASE not set');
+if (!process.env.PGDATABASE && !process.env.DATABASE_URL) {
+  throw new Error("PGDATABASE or DATABASE_URL not set");
+}
+
+const config = {};
+
+if (ENV === "production") {
+  config.connectionString = process.env.DATABASE_URL;
+  config.max = 2;
 }
 
 module.exports = new Pool();
```

Comment:
If I am not mistaken the `config` variable should be given to the `Pool` as an argument when exporting such as
```
module.exports = new Pool(config);
```

Double check this in the instructions in case my knowledge is out of date